src/main/java/io/github/thebusybiscuit/cscorelib2/protection/modules/LandsProtectionModule.java

```diff
@@ -36,7 +37,8 @@ public boolean hasPermission(OfflinePlayer p, Location l, ProtectableAction acti
         LandWorld landWorld = landsIntegration.getLandWorld(l.getWorld());
```

Comment:
If this is supposed to support OfflinePlayers now, then you will of course have to remove the instanceof - check  in the line above (cannot mark that one as it wasn't changed by this PR)